pkg/rke2/psp_templates.go

```diff
@@ -33,11 +33,6 @@ const globalRestrictedPSPTemplate = `apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
```

Comment:
This policy looks like it came from the second example policy here:  https://kubernetes.io/docs/concepts/policy/pod-security-policy/#example-policies

If so, that has a comment in it that says: 
>     # This policy assumes the nodes are using AppArmor rather than SELinux.
but we are pulling out all the AppArmor stuff, which would appear to leave it a bit toothless. I don't have a suggested alternative, but should we perhaps go with a completely different PSP?

Reply:
We have a flag for selinux, may we need one for apparmor and a combination could drive the psp template?

This will also mean that user will have more control on how the behaviour of the psp.

Reply:
Sounds like CIS doesn't care either way, so we're fine to just drop the annotations and leave the seLinux rule open.